File: Wei.Web/app/model/setting/ModuleType.js
Ext.define('Wei.model.setting.ModuleType', {
    extend: 'Wei.model.Base',

    fields: [
        { type: 'int', name: 'id', persist: true },
        { type: 'string', name: 'name', persist: true },
        { type: 'string', name: 'remark', persist: true }
    ]
});

File: Wei.Core/Domain/Questions/ModuleType.cs
namespace Wei.Core.Domain.Questions
{
    public class ModuleType: BaseEntity
    {
        /// <summary>
        /// 模块类型名称
        /// </summary>
        public string Name { get; set; }
        /// <summary>
        /// 模块备注
        /// </summary>
        public string Remark { get; set; }
    }
}
